src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -99,10 +124,17 @@ class HuiMapCard extends LitElement implements LovelaceCard {
     ) {
       throw new Error("Parameter geo_location_sources needs to be an array");
     }
-
-    this._config = config;
-    this._configEntities = config.entities
-      ? processConfigEntities<MapEntityConfig>(config.entities)
+    if (config.show_all && (config.entities || config.geo_location_sources)) {
+      throw new Error(
+        "Cannot specify show_all and entities or geo_location_sources"
```

Comment:
I considered if this should be allowed (like if you want to show_all _and_ additionally apply some custom config to a specific entity), but it got a little confusing with the mix of entities and geo_sources, and I'm not sure if there is demand for this, so I punted for now for simplicity.